src/common/tests/common_test.h

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2019 Intel Corporation.
+ * (C) Copyright 2019-2020 Intel Corporation.
```

Comment:
Technically doesn't seem required.

Reply:
Probably happened due to a merge commit where someone didn't update it.   This happens to me quite a lot but hopefully less if people are installing the hook.